src/main.cpp

```diff
@@ -40,7 +40,7 @@
 
 #include "binding.h"
 
-#ifdef WIN32 || WIN64
+#if defined(WIN32) && defined(WIN64)
```

Comment:
I've used `__WINDOWS__` elsewhere in mkxp before. Could you check if that works for both architectures?